module1/lessons/flow_control.md

```diff
@@ -345,8 +448,38 @@ loop do
 end
 ```
 
-If you accidentally get stuck in an infinite loop, use `control + c` to stop it.
+</section>
+
+## Looping in JavaScript
```

Comment:
Although they're not as popular, doesn't JS also have while loops? I wonder if we should mention that these are available, but the for loop is more common

Reply:
Happy to leave this for now since for loops are more popular.